.gitignore

```diff
@@ -1 +1,2 @@
-__pycache__/
\ No newline at end of file
+__pycache__/
+stan_test.py
```

Comment:
I have a local `stan_test.py` file that I can quickly run to test out random stuff in Python. Adding it to the `.gitignore` because it shouldn't be part of the commits.